File: Lesson5 Sorting/BubbleSort.py
array = [100, 234, 23, 1230, 90, 234, 44, 21, 420];
isSorted = False;
while isSorted == False:
    isSorted = True;
    for i in range (len(array) - 1):
        if(array[i] > array[i + 1]):
            temp = array[i]
            array[i] = array[i + 1]
            array[i + 1] = temp
            isSorted = False;

for i in array:
    print i,

File: Lesson5 Sorting/BubbleSort.java
public class BubbleSort {
	public static void main(String[]args) {
		int array[] = {1, 39, 2034, 432, 43, 30, 2};
		boolean isSorted = false;
		while(!isSorted) {
			isSorted = true;
			for(int i = 0; i < array.length - 1; i++) {
				if(array[i] > array[i + 1]) {
					int temp = array[i];
					array[i] = array[i + 1];
					array[i + 1] = temp;
					isSorted = false;
				}
			}
		}
		for(int i = 0; i < array.length; i++) {
			System.out.print(array[i] + " ");
		}
	}
}
